test/qtiCommonRenderer/interactions/textEntry/test.js

```diff
@@ -153,7 +145,7 @@ define([
                 var $input = $container.find('.qti-interaction.qti-textEntryInteraction');
                 assert.equal(
                     getTooltipContent($input),
-                    __('This is not a valid answer'),
+                    undefined,
```

Comment:
`assert.ok(!getTooltip($input), 'the error tooltip is hidden after a correct response');` maybe better?

Reply:
I will change, but with another manner, see the response of comment above